packages/traceloop-sdk/tests/test_prompt_management.py

```
@@ -87,3 +238,30 @@ def test_prompt_management(exporter, openai_client):
+def test_prompt_management_with_response_format(exporter, openai_client):
+    PromptRegistryClient()._registry.load(prompts_json=json.loads(prompts_with_response_format_json))
+    prompt_args = get_prompt(key="joke_generator", variables={"style": "pirate"})
+    openai_client.chat.completions.create(**prompt_args)
+
+    spans = exporter.get_finished_spans()
+    open_ai_span = spans[0]
+    completion = open_ai_span.attributes.get(f"{SpanAttributes.LLM_COMPLETIONS}.0.content")
+    try:
+        json.loads(completion)
+    except json.JSONDecodeError:
+        assert False
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)